pytket/binders/circuit/Circuit/add_classical_op.cpp

```diff
@@ -194,6 +196,26 @@ void init_circuit_add_classical_op(
             return add_gate_method<Bit>(&circ, op, args, kwargs);
           },
           "See :py:meth:`add_c_setbits`.", py::arg("values"), py::arg("args"))
+      .def(
+          "add_c_setreg",
+          [](Circuit &circ, const _tket_uint_t value, const BitRegister &reg,
+             const py::kwargs &kwargs) {
+            auto bs = std::bitset<_TKET_REG_WIDTH>(value);
+            std::vector<Bit> args(reg.size());
+            std::vector<bool> vals(reg.size());
+            for (unsigned i = 0; i < reg.size(); i++) {
+              args[i] = reg[i];
+              vals[i] = bs[i];
+            }
+
+            std::shared_ptr<SetBitsOp> op = std::make_shared<SetBitsOp>(vals);
+            return add_gate_method<Bit>(&circ, op, args, kwargs);
+          },
+          "Set a classical register to an unsigned integer value. The "
+          "little-endian "
+          "bitwise representation of the integer is truncated to the register "
+          "size, up to 32 bit width.",
```

Comment:
How do you suggest doing this, `sprintf`? I can't find a simple way (c++ 20 has `std::format`...)

Reply:
Actually the easiest way might be to make `_TKET_REG_WIDTH` a `#define` -- it might as well be a compile-time constant -- and just do
```suggestion
          "size, up to " ## #_TKET_REG_WIDTH ## " bit width.",
```